app/models/Reservation.java

```diff
@@ -41,6 +44,7 @@
      */
 
     @Override
+    @JsonView(View.AdditionalDetails.class)
```

Comment:
why is this method parsed in any case? it should be ignore... same for all cases of annotating helper methods with JsonView